modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraAssignmentsRepository.java

```diff
@@ -482,9 +482,9 @@ public void assignUsersInBatch(List<Pair<Experiment, Assignment>> assignments, D
      */
     private void incrementCounts(List<Pair<Experiment, Assignment>> assignments, Date date) {
         boolean countUp = true;
-        assignments.forEach(pair -> {
-            assignmentsCountExecutor.execute(new AssignmentCountEnvelope(this, experimentRepository, dbRepository, pair.getLeft(), pair.getRight(), countUp, eventLog, date, assignUserToExport, assignBucketCount));
-        });
+        assignments.forEach(pair -> assignmentsCountExecutor.execute(new AssignmentCountEnvelope(
```

Comment:
@bargenilesh There was a change earlier. I reverted it in the latest change but as I touched this class, the original code was going way above the monitor width, therefore I split it into 2 lines for better readability. The code formatter does not complain even after the split, so should be fine, I guess! :)